package.json

```diff
@@ -26,6 +26,7 @@
     "@docusaurus/core": "2.0.0-alpha.70",
     "@docusaurus/plugin-google-gtag": "2.0.0-alpha.70",
     "@docusaurus/preset-classic": "2.0.0-alpha.70",
+    "@docusaurus/theme-classic": "^2.0.0-beta.3",
```

Comment:
All @ docusaurus packages should have the same version. This might cause issues.